modules/persistent-storage-csi-cloning-overview.adoc

```diff
@@ -19,6 +19,6 @@ No new API objects are required for cloning. The existing `dataSource` field in
 By default, {product-title} supports CSI volume cloning with these limitations:
 
 * The destination persistent volume claim (PVC) must exist in the same namespace as the source PVC.
-* The source and destination storage class must be the same.
+* You can use a different storage class as a source and the same volume as a destination for CSI volume cloning. Alternatively, You can use the default storage class as a source and omit the `storageClassName` in the `spec`.
```

Comment:
You can use a different storage class as a source and the same volume as a destination.

This is not accurate to me, the destination volume can be the same or a different storage class as the source, there should not mention "the same volume". 

How about: 
```
Cloning is supported with a different Storage Class.
    Destination volume can be the same or a different storage class as the source.
    Default storage class can be used and storageClassName omitted in the spec.
```

Reply:
@duanwei33 Updated the text. Addressed